pkg/oc/admin/prune/images.go

```diff
@@ -104,19 +106,24 @@ var (
 	defaultKeepYoungerThan         = 60 * time.Minute
 	defaultKeepTagRevisions        = 3
 	defaultPruneImageOverSizeLimit = false
+
+	// TODO choose final name and promote to API
+	protectionLabel = "image.openshift.io/allow-prune-immunity"
```

Comment:
Can it be more general? (e.g. `openshift.io/allow-prune-immunity`). We have pruners for resources other than image.

Reply:
> Can it be more general? (e.g. openshift.io/allow-prune-immunity). We have pruners for resources other than image.

I would rather start specific and move out to generality.  Pruning isn't a well explored space yet.  See how deployments do it as a counter example.